pom.xml

```diff
@@ -253,4 +259,16 @@
     <module>app</module>
     <module>benchmarks</module>
   </modules>
+
+  <profiles>
+    <profile>
+      <id>serverTest</id>
+      <properties>
+        <skipRunTests>true</skipRunTests>
+      </properties>
+      <modules>
+        <module>server</module>
+      </modules>
+    </profile>
+  </profiles>
```

Comment:
Shall we create a profile for each module in order to use it for running module target instead of using `-pl`?

Reply:
In this case, we need separate `skipRunTests` parameter for each module, I can do, but I don't know if it worths to add them, I'd vote to add separate profiles as needed, since we sparsly work on other things (like extensions/*, plugins/* etc.), what do you think?

Reply:
+1 for doing it only for server. Would be good if we could avoid bloating up the build configuration 

Instead, we could consider bringing this topic up upstream and see if they're open to address this in a general way - or at least learn why it is the way it is